sharepoint/sharepoint-ps/sharepoint-online/Get-SPOBuiltInFontPackageSettings.md

```diff
@@ -0,0 +1,36 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spobuiltinfontpackagesettings
+applicable: SharePoint Online
+title: Get-SPOBuiltInFontPackageSettings
+schema: 2.0.0
+author: yixianpu
+ms.author: yixianpu
+ms.reviewer:
+---
+
+# Get-SPOBuiltInFontPackageSettings
+
+## SYNOPSIS
+
+Gets settings of built-in font packages.
```

Comment:
I feel this could be improved a little bit, e.g., to add a bit more info? I'd suggest you work with your PM on this PR.

Reply:
(applies to the other file too)